fabfile/db/__init__.py

```diff
@@ -32,12 +32,14 @@
 settings too gives it standart value.
 If you want to use User and Password not the same as in settings you can type them
 in command line.
+If you want to use Database Name not the same as in settings you can type it
+in command line.
```

Comment:
Add plz that this custom params (host, port etc.) are not permanent and will not be saved in settings.